includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -463,17 +474,21 @@ public function getData() {
 	}
 
 	private function getSitenameData() {
-		$wgSitenameForComscoreForWikiId = WikiFactory::getVarValueByName( 'wgSitenameForComscore', $this->wikiId );
-
-		if ( $wgSitenameForComscoreForWikiId ) {
-			$sitename = $wgSitenameForComscoreForWikiId;
+		if ( $this->product === self::FANDOM_PRODUCT ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)